EasyPost/Parameters/Tracker/CreateList.cs

```diff
@@ -1,3 +1,4 @@
+using System;
```

Comment:
question: didn't we remove the function this parameter set is used for? If so we should also clean this up since it cannot be used.

Reply:
We did, it's being removed in a separate PR of breaking changes.